### PR TITLE
Command add-meeting tests

### LINKED ISSUE #60: Add tests for add-meeting command

### PR DESCRIPTION
**Pull request by ivantjh**
- Set up some infrastructure for `Event` testing
- Add tests for add-meeting command

Fixes #60

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 1]
```diff
@@ -68,4 +72,12 @@ public int hashCode() {
         return Objects.hash(feedbackToUser, showHelp, exit);
     }
 
+    @Override
+    public String toString() {
+        return "CommandResult{"
+                + "feedbackToUser='" + feedbackToUser + '\''
+                + ", showHelp=" + showHelp
+                + ", exit=" + exit
+                + '}';
```
Comment (chesterhow) [inline src/main/java/seedu/address/logic/commands/CommandResult.java]: Let's standardise to double quotes for all strings

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -81,8 +81,18 @@ public static Picture getPicture() {
...
     }
 
+    public static List<Event> getMeetingList(Event... meetings) {
+        return Arrays.stream(meetings)
+                .collect(Collectors.toList());
+    }
+
     public static List<Event> getDateList() {
         return Collections.emptyList();
     }
```
Comment (chesterhow) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:91]: Should be okay to remove the old `getMeetingList()` and `getDateList()` which returns empty lists right?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -83,6 +83,8 @@ public static void assertCommandSuccess(Command command, Model actualModel, Comm
             Model expectedModel) {
         try {
             CommandResult result = command.execute(actualModel);
+            System.out.println("result");
+            System.out.println(result);
```
Comment (chesterhow) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: I think this can be removed?

**Review by chesterhow — APPROVED** [review]
LGTM, just some minor nits.

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 2 above)
Comment (ivantjh) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:91]: Nope, this is still used when the `Person` being initialised has no meetings and no dates.

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 3 above)
Comment (ivantjh) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: yeap, my bad

**Pull request merged**